src/ui.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "ux.h"
+
+void ui_idle_flow(void);
+void ui_sign_flow(void);
```

Comment:
The name `ui_sign_flow` seems a bit ambiguous to me. I would rather call it `ui_sign_message_flow` or `ui_sign_msg_flow` to make it more explicit.